spec/presenters/tree_builder_spec.rb

```diff
@@ -60,4 +60,55 @@
       expect(nodes).to be_empty
     end
   end
+
+  # This is testing a private method, but it's relied upon by a lot of
+  # subclass methods, so it doesn't seem unreasonable to specify its
+  # behavior directly.
+  context '#count_only_or_objects' do
```

Comment:
Not necessary to change, but usually I put "(private method)" or "(private)" at the end of the context string to show that it's a private method.